packages/saber-plugin-image/lib/index.js

```diff
@@ -0,0 +1,92 @@
+const { join } = require('path')
+const { parse } = require('querystring')
+
+const ID = 'images'
+
+exports.name = ID
+
+exports.apply = (api, options = {}) => {
+  options = Object.assign(
+    {
+      lazyLoad: true,
+      placeholder: true,
+      blendIn: true,
+      markdownImages: false
```

Comment:
Maybe it makes sense to enable this by default? If you use this plugin, it's very likely that you want it for markdown images as well 🤔

Reply:
Probaby, yeah.